src/main/java/spoon/support/util/SortedList.java

```diff
@@ -42,9 +42,9 @@ public void add(int index, E element) {
 
 	@Override
 	public boolean addAll(Collection<? extends E> c) {
-		boolean ret = true;
+		boolean ret = false;
 		for (E e : c) {
-			ret &= add(e);
+			ret |= add(e);
```

Comment:
As we'll always return `true` this way (because `add` always returns true), we can probably also just return `!c.isEmpty()` (assuming `c` doesn't get modified in between) or just do `ret = true`, similar to the default implementation in `AbstractList`. What do you think? (Seems like the Qodana inspection is a bit aggressive here...)

Reply:
Qodana is right i didn't notice it was using bit-wise operations instead of logical operations i have fixed that now

Reply:
I hope the PR title is okay now

Reply:
i hope that the title is okay now... trying to follow the procedures that indicate that fic and the number of the issue needs to be present too

Reply:
Ping?

Reply:
Please use one of my suggested variants or your initial version instead of the `||`. The title is fine now.

Reply:
Okay, i've reverted the change to use bit-wise logic again.